src/osal/common/osal_helpers.c

```diff
@@ -0,0 +1,42 @@
+ /*********************************************************************
+ *        _       _         _
+ *  _ __ | |_  _ | |  __ _ | |__   ___
+ * | '__|| __|(_)| | / _` || '_ \ / __|
+ * | |   | |_  _ | || (_| || |_) |\__ \
+ * |_|    \__|(_)|_| \__,_||_.__/ |___/
+ *
+ * www.rt-labs.com
+ * Copyright 2018 rt-labs AB, Sweden.
+ *
+ * This software is dual-licensed under GPLv3 and a commercial
+ * license. See the file LICENSE.md distributed with this software for
+ * full license information.
+ ********************************************************************/
+
+#include <osal.h>
+
+#include <stdio.h>
+
+ void os_ip_to_string(
```

Comment:
Code that doesn't need porting to the OS should obviously not be in the OS porting layer. Move these convenience functions to the demo (or stack if it also needs them?)

Reply:
Done